src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl_IVsSolutionLoadEvents.cs

```diff
@@ -23,6 +23,7 @@ internal partial class VisualStudioWorkspaceImpl : IVsSolutionLoadEvents
     {
         int IVsSolutionLoadEvents.OnBeforeOpenSolution(string pszSolutionFilename)
         {
+            GetProjectTrackerAndInitializeIfNecessary(ServiceProvider.GlobalProvider).OnBeforeOpenSolution();
```

Comment:
Do we want to force create it here, or leave it null if it's the first solution being opened?